.github/actions/docker/entrypoint.sh

```diff
@@ -9,19 +9,29 @@ event_name=$4
 tag=$5
 dockerhub_user=$6
 dockerhub_password=$7
+build_args=$8
 
 echo "Log in to registry."
 echo $password | docker login -u ${username} --password-stdin docker.pkg.github.com || exit 1
 
-echo "Building image."
-docker build docker -t docker.pkg.github.com/${username}/${project}/${image}:${tag} -f docker/Dockerfile-${image} || exit 1
+full_tag=docker.pkg.github.com/${username}/${project}/${image}:${tag}
+build_command="docker build docker -t ${full_tag} -f docker/Dockerfile-${image}"
+
+if [ "$build_args" != "" ]; then
+    for arg in $build_args; do
+        build_command="${build_command} --build-arg ${arg}"
+    done
+fi
+
+echo "Building image with tag: ${full_tag}"
+eval $build_command || exit 1
```

Comment:
I'd rather not have `eval` in shell scripts. The if/else above is also not necessary, for loops on empty strings are not evaluated. After removing both in 187d3efce, the build still seems to work:
```sh
# fedora:
"fedora" "jngrad/espresso-docker" "***" "push" "187d3efce117f30b6cb4d39b76a9778851e7f800" "" "" ""
Building image with tag: docker.pkg.github.com/jngrad/espresso-docker/fedora:187d3efce117f30b6cb4d39b76a9778851e7f800
# fedora-latest:
"fedora" "jngrad/espresso-docker" "***" "push" "latest_base" "" "" "BASE_VERSION=latest"
Building image with tag: docker.pkg.github.com/jngrad/espresso-docker/fedora:latest_base
```

Reply:
> I'd rather not have eval in shell scripts
 
why?

Reply:
`eval` is prone to accidental issues with quotes, it's not obvious what the string looks like when passed through for loops and conditionals, and can lead to other types of issues that can be difficult to diagnose (https://stackoverflow.com/a/17529221).